House-of-glass/src/App.css

```diff
@@ -61,4 +61,85 @@
     grid-template-rows: 15rem auto 12.5rem;
     width: 90%;
   }
+}
+
+.products-container{
+
+  display: grid;
+  grid-template-columns: 1fr auto;
+  grid-template-areas: 
+  "filter text "
+  "filter products"
+  "filter products";
+
+}
+
+.products-text-container{
+  grid-area: text;
+
+}
+
+/* Filter Pills */
+.pill{
+  border: 2px solid black;
+  width: fit-content;
+  padding: 0.2rem 0.5rem;
+  border-radius: 40px;
+  margin-bottom: 20px;
+}
+
+.category-wrapper{
+  display: flex;
+  outline: 2px solid black;
+  border-radius:16px;
+  padding: 2rem 1rem 0rem;
+  height:fit-content;
+  width:100%;
+  flex-direction: column;
+  justify-content:flex-start;
+  position: relative;
+  margin-bottom: 2rem;
+}
+
+.category-label-container{
+  position: absolute;
+  top:-20px;
+  background-color: white;
+  border-radius:30px;
+}
+
+.filter{
+  grid-area: filter;
+  min-width: 20vw;
```

Comment:
defining widths like this causes hindrance in making site responsive

Reply:
Yes, My bad. I've shifted it to 20rem for the time being. Thank You.